rmgpy/pdep/networkTest.py

```diff
@@ -194,12 +195,48 @@ def test_netReactions(self):
         """
         self.assertEqual(0, len(self.network.netReactions))
     
+    @work_in_progress
     def test_initialize(self):
         """
         Test that the Network.initialize() method.
         """
         self.network.initialize(Tmin=300., Tmax=2000., Pmin=1e3, Pmax=1e7, minimumGrainCount=200, maximumGrainSize=4184.0)
-    
+        self.assertTrue(False,'This test was never finished being written')
```

Comment:
Although unfinished, what is the purpose of this test? Just to see an object was created, and maybe access two or three attributes? If it's that straight forward, lets add this and make it functional. Otherwise, it could also be OK to delete this unit test (from a quick search, this is the only `test_initialize` in the repo).

Reply:
its fine with me to delete it.

Reply:
Agree